src/lang-promql/complete/promql.terms.ts

```diff
@@ -17,67 +19,251 @@ export const binOpTerms = [
+  },
+  {
+    label: 'rate',
+    info: 'Calculate per-second increase over a range vector (for counters)',
+  },
+  {
+    label: 'resets',
+    info: 'Return number of value decreases (resets) in input series of time',
+  },
+  {
+    label: 'round',
+    info: 'Round values of input series to nearest integer',
+  },
+  {
+    label: 'scalar',
+    info: 'Convert single-element series vector into scalar value',
+  },
+  {
+    label: 'sort',
+    info: 'Sort input series ascendingly by value',
+  },
+  {
+    label: 'sort_desc',
+    info: 'Sort input series descendingly by value',
+  },
+  {
+    label: 'sqrt',
+    info: 'Return the square root for input series',
+  },
+  {
+    label: 'stddev_over_time',
+    info: 'Calculate the standard deviation within input series over time',
+  },
+  {
+    label: 'stdvar_over_time',
+    info: 'Calculate the standard variation within input series over time',
+  },
+  {
+    label: 'sum_over_time',
+    info: 'Calculate the sum over the values of input series over time',
+  },
+  {
+    label: 'time',
+    info: 'Return the Unix timestamp at the current evaluation time',
+  },
+  {
+    label: 'timestamp',
+    info: 'Return the Unix timestamp for the samples in the input vector',
+  },
+  {
+    label: 'vector',
+    info: 'Convert a scalar value into a single-element series vector',
+  },
+  {
+    label: 'year',
+    info: 'Return the year for provided timestamps',
+  },
 ];
-export const aggregateOpTerms = [
-  { label: 'avg' },
-  { label: 'bottomk' },
-  { label: 'count' },
-  { label: 'count_values' },
-  { label: 'group' },
-  { label: 'max' },
-  { label: 'min' },
-  { label: 'quantile' },
-  { label: 'stddev' },
-  { label: 'stdvar' },
-  { label: 'sum' },
-  { label: 'topk' },
+functionIdentifierTerms.forEach((term) => {
+  term.detail = 'function';
+});
+export const aggregateOpTerms: AutoCompleteNode[] = [
+  {
+    label: 'avg',
+    info: 'Calculate the average over dimensions',
+  },
+  {
+    label: 'bottomk',
+    info: 'Smallest k elements by sample value',
+  },
+  {
+    label: 'count',
+    info: 'Count number of elements in the vector',
+  },
+  {
+    label: 'count_values',
+    info: 'Count number of elements with the same value',
+  },
+  {
+    label: 'group',
+    info: 'All values in the resulting vector are 1',
+  },
+  {
+    label: 'max',
+    info: 'Select maximum over dimensions',
+  },
+  {
+    label: 'min',
+    info: 'Select minimum over dimensions',
+  },
+  {
+    label: 'quantile',
+    info: 'Calculate φ-quantile (0 ≤ φ ≤ 1) over dimensions',
+  },
+  {
+    label: 'stddev',
+    info: 'Calculate population standard deviation over dimensions',
+  },
+  {
+    label: 'stdvar',
+    info: 'Calculate population standard variance over dimensions',
+  },
+  {
+    label: 'sum',
+    info: 'Calculate sum over dimensions',
+  },
+  {
+    label: 'topk',
+    info: 'Largest k elements by sample value',
+  },
 ];
+aggregateOpTerms.forEach((term) => {
```

Comment:
It would be nicer to not mutate the existing array, but to just assign the final one like:

```typescript
export const aggregateOpTerms: AutoCompleteNode[] = [
  // ...
].map((term) => {...term, detail: 'aggregation'});
```

Reply:
I would more suggest to duplicate the label detail everywhere. It's stupid but simpler and clearer

Reply:
Also fine with me.

Reply:
Thank you. Fixed in a5aa063ad5a613cd23c50be6a0cc9e795ccca57b